components/places/android/build.gradle

```diff
@@ -63,6 +63,9 @@ cargo {
     // The Cargo targets to invoke.  The mapping from short name to target
     // triple is defined by the `rust-android-gradle` plugin.
     targets = [
+        'linux-x86-64',
```

Comment:
I was chatting with @ncalexan about testability of this stuff the other day... Perhaps he'll have an opinion here?

Reply:
This looks good.  We should have been building these targets from the beginning.  I have changes slated for the `forUnitTests` variants; I'll try to make this uniform while I'm there.